### PR TITLE
Pull request for issue #143 #team2

### PR DESCRIPTION
**Pull request by ghost**
#143: Corrected registration bug.

### CODE REVIEW

File: piplmesh/account/form_fields.py [hunk 1]
```diff
@@ -27,7 +27,8 @@ def clean(self, value):
 
         def error():
             raise forms.ValidationError(self.error_messages['bounds'])
-
-        fields.limit_date(value, self.lower_limit, self.upper_limit, error)
+        
+        if value != None:
```
Comment (mitar) [inline piplmesh/account/form_fields.py]: Checking for `None` value should be always done with `is`:

```
if value is not None:
```

Anyway, wouldn't it be better to test this in `limit_date` function? If `value` is `None` it returns immediately?

**Comment by ghost** [comment]
It's better?

File: piplmesh/account/fields.py [hunk 2]
```diff
@@ -11,6 +11,9 @@
 )
 
 def limit_date(value, lower_limit, upper_limit, error):
+    if value is None:
```
Comment (mitar) [inline piplmesh/account/fields.py]: In this case it is probably enough:

```
if not value:
    return
```

File: piplmesh/account/form_fields.py [hunk 3]
```diff
@@ -27,7 +27,7 @@ def clean(self, value):
 
         def error():
             raise forms.ValidationError(self.error_messages['bounds'])
-
+
```
Comment (mitar) [inline piplmesh/account/form_fields.py]: Why change here?

File: piplmesh/account/fields.py (hunk 2 above)
Comment (ghost) [inline piplmesh/account/fields.py]: Corrected condition.

File: piplmesh/account/form_fields.py (hunk 3 above)
Comment (ghost) [inline piplmesh/account/form_fields.py]: Sorry, my mistake. :)

Comment (mitar) [inline piplmesh/account/form_fields.py]: No problem, fix it. :-)

Comment (ghost) [inline piplmesh/account/form_fields.py]: Done.

**Pull request merged**